library/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "module": "ESNext",
     "moduleResolution": "node",
     "allowImportingTsExtensions": true,
-    "noEmit": true
+    "noEmit": true,
+    "forceConsistentCasingInFileNames": true
```

Comment:
I think that `forceConsistentCasingInFileNames` is `true` by default. There is no reason to add it explicitly or?

Reply:
Interesting, Microsoft's Webhint (https://webhint.io/docs/user-guide/hints/hint-typescript-config/consistent-casing/), mentioned it is off, but https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames says it's on by default. Maybe on since TS 5? Not sure, but since I've bumped to latest TS in the other PR, I'll remove it again